lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -680,7 +693,7 @@ def __init__(self, service: _ServiceType, snap: bool = False) -> None:
         self._ops_manager = _SnapManager() if snap else _AptManager(service)
         self.service = self._ops_manager.service_manager_for(service)
         self.munge = _MungeManager(self._ops_manager)
-        self.jwt = _JWTKeyManager(snap)
+        self.jwt = _JWTKeyManager(self._ops_manager.var_lib_path / "slurm.data/jwt_hs256.key")
```

Comment:
nit: we could move this to `_JWTKeyManager` and only pass the manager, to have everything about jwt encapsulated in a single class.

Reply:
Sure thing, not a huge issue for me. This also reduces the chances that we need to modify the constructor for `_JWTKeyManager` in the future.